modules/react/combobox/lib/ComboboxMenuItem.tsx

```diff
@@ -57,17 +61,37 @@ export const useComboboxMenuItem = composeHooks(
   useListItemRegister,
   useListItemAllowChildStrings
 );
+const comboboxMenuItemIconStencil = createStencil({
```

Comment:
I'm wondering if this needs to be a stencil or not. 

If it's just one style I think we can use `createStyles` since we're using `mergeStyles`.